### PR TITLE
Tutorial: polish for release

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
- Fixes broken profile step
- Hide caret on mobile

### CODE REVIEW

**Comment by jyng** [comment]
I seem to have found some new bugs.

### Tutorial is stuck if I had exited via the 'X'
Repro:
1) Start the tutorial
2) Exit at any step. 
3) Confirm `End Tutorial `
4) Start tutorial again through Leap. I get 'End Tutorial Now?' again.



### If I do a hard refresh(clear my cookies) and I get prompted the initial 'Welcome to Landscape..'
Ideally, we should be prompted this screen only on the user's first entry. Do you think that would be reasonably possible? Checking to see if it's the user's first time booting and starting the ship?

### Tutorial restores state of where it left off
This was in issue about tutorial stopping on profile. But @urcades notes a different issue. I wasn't able to repro due the first issue I listed here.
https://github.com/urbit/landscape/issues/520#issuecomment-790891867

**Comment by matildepark** [comment]
> Ideally, we should be prompting this screen only on the user's first entry. Do you think that would be reasonably possible? Checking to see if it is the user's first time booting?

I mean, that's why we use `settings-store` here, it's meant to be once per boot. If you didn't wrap up the tutorial and we didn't mark it as 'seen' it would show this modal next time presumably.

**Comment by liam-fitzgerald** [comment]
Yeah it'll only mark as done when you've either a) dismissed the tutorial or b) finished it

File: pkg/interface/src/views/apps/profile/components/Profile.tsx [hunk 1]
```diff
@@ -160,43 +161,38 @@ export function Profile(props: any): ReactElement {
 
   const anchorRef = useRef<HTMLElement | null>(null);
 
-  useTutorialModal('profile', ship === `~${window.ship}`, anchorRef);
+  console.log(ship);
```
Comment (matildepark) [inline pkg/interface/src/views/apps/profile/components/Profile.tsx:164]: *whispers* console.log...

Comment (liam-fitzgerald) [reply pkg/interface/src/views/apps/profile/components/Profile.tsx:164]: sry!

**Comment by jyng** [comment]
> Yeah it'll only mark as done when you've either a) dismissed the tutorial or b) finished it

Hm, interesting. I was running on incognito mode and it prompted up even when I had finished the tutorial previously.

**Pull request merged**